pkgs/development/tools/operator-sdk/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildGoModule, go, lib, fetchFromGitHub, makeWrapper, ... }:
+
+buildGoModule rec {
+  name = "operator-sdk";
```

Comment:
```suggestion
  pname = "operator-sdk";
```

Reply:
I'm curious, what's the difference between `pname` and `name`?

Reply:
From [nixpkgs manual](https://nixos.org/nixpkgs/manual):
> For convenience, you can also use pname and version attributes and mkDerivation will automatically set name to "${pname}-${version}" by default. Since RFC 0035, this is preferred for packages in Nixpkgs, as it allows us to reuse the version easily